src/indexeddb-js.js

```diff
@@ -18,7 +18,7 @@
 if ( typeof(define) !== 'function')
   var define = require('amdefine')(module);
 
-define(['underscore'], function(_) {
+define(['underscore'], function(_) {'use strict';
```

Comment:
can you move this to a line by itself? i prefer that -- it makes it clearer that it is not part of the import definition.